src/time.js

```diff
@@ -9,6 +9,20 @@ function advanceBlock () {
   });
 }
 
+// Advance the block to the passed height
+async function advanceBlockTo (target) {
+  if (!BN.isBN(target)) {
+    target = new BN(target);
+  }
+
+  let now = (await latestBlock());
```

Comment:
I'd rename this from `now` to something like `current`, or perhaps event `currentBlock`, to make it clear that this is not a timestamp.